tests/integration/suite/test_workloads.py

```diff
@@ -310,6 +310,34 @@ def test_statefulset_workload_volumemount_subpath(admin_pc):
         assert e.value.error.status == 422
 
 
+def test_workload_redeploy(admin_pc, remove_resource):
+    client = admin_pc.client
+    ns = admin_pc.cluster.client.create_namespace(
+        name=random_str(),
+        projectId=admin_pc.project.id)
+    remove_resource(ns)
+    name = random_str()
+    workload = client.create_workload(
+        name=name,
+        namespaceId=ns.id,
+        scale=1,
+        containers=[{
+            'name': 'one',
+            'image': 'nginx',
+        }],
+        annotations={
+            "cattle.io/timestamp":
+                datetime.datetime.now().isoformat()}, )
+    remove_resource(workload)
+    workload = wait_for_workload_active(client, workload)
+    client.action(workload, "redeploy")
+    timestamp = client.list_workload(
+        uuid=workload.uuid).data[0].annotations["cattle.io/timestamp"]
+    assert timestamp != workload.annotations["cattle.io/timestamp"]
+    client.delete(workload)
+    admin_pc.cluster.client.delete(ns)
```

Comment:
Both of these delete calls are unnecessary, `remove_resource` takes care of this and will handle errors.